sentinel/tests/unit/transitions/accept_patient_reports.js

```diff
@@ -196,47 +196,295 @@ describe('accept_patient_reports', () => {
       });
     });
 
-    it('adds report_uuid property', done => {
+    // Helpful diagram for the next 5 tests:
+    // https://github.com/medic/medic/issues/4694#issuecomment-459460521
+    it('does not associate visit to anything since no reminder messages have been sent yet', done => {
+      sinon.stub(transition, '_silenceReminders').callsArgWith(3, null, true);
+      const doc = {
+        _id: 'z',
+        fields: { patient_id: 'x' },
+        reported_date: '2018-09-17T18:45:00.000Z',
+      };
+      const config = { silence_type: 'x', silence_for: '8 days', messages: [] };
+      const registrations = [
+        {
+          _id: 'a',
+          reported_date: '2017-02-05T09:23:07.853Z',
+          scheduled_tasks: [
+            {
+              due: '2018-09-28T19:45:00.000Z',
+              state: 'scheduled',
+              messages: [
+                {
+                  uuid: 'k5',
+                },
+              ],
+              group: 1
+            },
+            {
+              due: '2018-10-28T20:45:00.000Z',
+              state: 'scheduled',
+              messages: [
+                {
+                  uuid: 'k',
+                },
+              ],
+              group: 1
+            },
+            {
+              due: '2018-11-28T21:45:00.000Z',
+              state: 'scheduled',
+              messages: [
+                {
+                  uuid: 'j',
+                },
+              ],
+              group: 1
+            },
+          ],
+        },
+      ];
+      sinon.stub(utils, 'getReportsBySubject').resolves(registrations);
+      transition._handleReport(doc, config, (err, complete) => {
+        complete.should.equal(true);
+        (typeof registrations[0].scheduled_tasks[0].responded_to_by).should.equal('undefined');
```

Comment:
using a wrapped typeof here is certainly novel, but kind of weird :P

The way we generally deal with it is to:
```js
const should = require('chai').should;
...
should.not.exist(...);
```

Reply:
Thanks! Note that it is `const should = require('chai').should();`